governance/reviews/template.md

```diff
@@ -23,171 +11,207 @@ Exemplary: project has an extraordinary level of governance development and impl
+* Document adoption of the CNCF Code of Conduct
+* CNCF Code of Conduct is cross-linked from other governance documents.
 
-### Maintainer List(s)
+<!--- The CNCF CoC should be present in all appropriate places in the project, which 
+includes their Community or main repository, and public website at a minimum.  Other
+community and governance documentation should refer to the CoC; for example, role
+removal process should mention that people can be removed for CoC violations.  
+If the project is handling their own CoC reports rather than just sending them 
+directly to the CNCF CoCC, you need to check if this is adequately documented, including
+who receives the reports and how and where they're evaluated. --->
 
-<!--- Check the list of CNCF-level Maintainers for the project. Answer the following question about the project's maintainers; Are they current? Are all of the people listed as Maintainers current & frequent contributors to the project, either code or non-code as required by the governance documents? What's the level of employer diversity in the current list of maintainers? Are employer affiliations listed in the maintainers list file? --->
+### Subprojects
 
-The project's maintainer list(s) [are/are not] current. Individuals on the maintainer list [do/do not] appear to match the requirements of maintainership in accordance with the project's documented requirements. The maintainer affiliations (employers) reflect [Balanced/Unbalanced] diversity.
+* All subprojects, if any, are listed.
+* If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.
 
-<!--- Note balance may be achieved through standing bodies, decision making, and other documentation. It should ensure no single entity can control the project's direction without informed consensus of other authorized parties. --->
+<!--- If the project has subprojects, plugins, or other divisions define them here. For each, is ownership and operation of clearly described? Are any standing committees/teams fully described, including listing their members? Does it conform to, align, and is it within scope of the governance expectations of the project?  Does is list communications channels and standing meetings?
+A subproject audit is often very useful to projects because this part of things tends to 
+bit-rot quite a bit, and you can help them see where they need to update it. --->
 
-### Evolution
+The project includes the following sub-projects, plugins, and other notable divisions:
 
-<!--- How has the project's governance evolved over time?  Is the project steadily refining/advancing its governance as the project grows and resolves issues? --->
+| Area | Ownership and Operation | Communications | Project Alignment | Notes |
+|:-----|:-----------------------:|:---------------:|:------------------|:---|
+|*sub-project*| Complete/Partial/Missing | Complete/Partial/Other | Complete/Partial/Conflict | |
```

Comment:
so we have a requirement that subprojects are listed, should a link to that listing be included in the governance review in addition to the evaluation table?

Reply:
That should be elsewhere in the doc; I moved things around to match the ordering of the Incubating/Graduated application.

Reply:
Not quite following?  What are you suggesting to add here?

Reply:
in its simplest form:
The project maintains a listing of sub-projects located here: $LINK
The sub-projects, plugins, and other notable divisions are as follows:
$TABLE

Reply:
this would provide an referenceable link to items included, and could surface a mis-alignment with the listed subprojects and discovered sub-projects.

Reply:
Aha.  Yeah, that's good when the project has a list.  Most projects don't; usually their application is the first time they actually make a list.  I'll write it as a swap in the template.